packages/phrases/src/locales/en.ts

```diff
@@ -51,6 +51,7 @@ const errors = {
     email_not_exists: 'The email address has not been registered yet.',
     phone_not_exists: 'The phone number has not been registered yet.',
     identity_not_exists: 'The social account has not been registered yet.',
+    identity_exists: 'The social account has been registered.',
```

Comment:
per #221, let's add an issue to re-scope these phrases (e.g. put all `register` phrases together in an object) cc @darcyYe